app/src/main/AndroidManifest.xml

```diff
@@ -16,6 +16,21 @@
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
 
+    <!-- Protect access to AuthenticatorBridgeService using this custom permission.
+
+    Note that each build type uses a different value for knownCerts.
+
+    This in effect means that the only application that can connect to the release variant
+    AuthenticatorBridgeService is the release variant Bitwarden Authenticator app. Furthermore,
+    the only application that can connect to the debug variant of AuthenticatorBridgeService
+    is the debug variant of the BitwardenAuthenticator app. -->
```

Comment:
Maybe we can condense this to something like:

```kotlin
    This in effect means that the only application that can connect to the debug/release/etc variant
    AuthenticatorBridgeService is the debug/release/etc variant Bitwarden Authenticator app.  -->
```

Reply:
I like it! Updated.